packages/hsl-shared-components/src/ActionBar/ActionBarItem.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/primitives';
+import { withTheme } from 'styled-components';
+import Touchable from '../Touchable';
+import View from '../View';
+import { P } from '../Typography';
+import { IS_NATIVE, size } from '../utils';
+
+const getTextColor = (active, inverted, theme) => {
+  if (inverted && !active) {
+    return theme.colors.primary.hslWhite;
+  }
+  if (inverted && active) {
+    return theme.font.colors.highlight;
+  }
+  if (active) {
+    return theme.colors.primary.hslWhite;
+  }
+  return theme.font.colors.highlight;
+};
+
+const getIconColor = (active, inverted, theme, override) => {
+  if (override && !active) {
```

Comment:
override won't override if active is true?

Reply:
Yes that's correct, setting something as active needs to changes background color.
But you're right, maybe we should give the choice to progframmer – he'd just need to supply us with different icon for active state if its needed.